code/style.css

```diff
@@ -9,50 +11,141 @@ p {
   background-color: #fff;
 }
 
+.header-styling {
```

Comment:
I understand what it means but I would choose another naming convention on the class here, maybe just `.header`?

Reply:
`styling` could be omitted here, indeed